web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx

```diff
@@ -73,12 +73,14 @@ function Modal({ ...props }) {
       <PopupModalContent>
         <div className="pb-10">
           <div className="flex flex-row items-center mas-buttons mb-3">
-            <RadioButton
-              defaultChecked={true}
-              onClick={() => handlePresetGas()}
-              {...presetArgs}
-            />
-            <p>{Intl.t('sendcoins.preset')}</p>
+            <div className="flex flex-row gap-3">
+              <RadioButton
+                defaultChecked={true}
+                onClick={() => handlePresetGas()}
+                {...presetArgs}
+              />
+              <p className="h-full text-center">{Intl.t('sendcoins.preset')}</p>
+            </div>
```

Comment:
This should be enough, no!?
 
```suggestion
              <RadioButton
                defaultChecked={true}
                onClick={() => handlePresetGas()}
                {...presetArgs}
              />
              <p className="h-full text-center ml-3">{Intl.t('sendcoins.preset')}</p>
            ```

Reply:
just need to add padding bottom to it so it centers the text